docs/UserGuide.md

```diff
@@ -18,14 +18,16 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
 2. Type the command in the command box.
 
 3. Press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+
 - Nav
   - `nav`: Navigates to the top-most context (root)
   - `nav {module_code / lecture_name}`: Navigates relative to the current context to a module or lecture context
   - `nav /mod {module_code / lecture_name} [/lec {lecture_name}]`: Navigates directly to the specified module or lecture context
   - `navb`: Navigates to the parent context of the current context
 - List
-  - `list /module`: Lists the names of all the recorded modules
-  - `list /module /{module_code}`: Lists all the videos recorded in that module code
+  - `list`: Lists the names of all the recorded modules
+  - `list /mod {module_code}`: Lists all the lectures in the specified module
+  - `list /mod {module_code} /lec {lecture_name}`: Lists all the videos in the specified module and lecture
```

Comment:
`/mod {module_code}` and `/lec {lecture_name}` should be specified as optional arguments as they are auto injected by the context sensitive system if not specified. Can refer to `add` command user guide documentation.